src/test/ui/regions/regions-close-over-type-parameter-1.nll.stderr

```diff
@@ -1,20 +0,0 @@
-error[E0310]: the parameter type `A` may not live long enough
```

Comment:
Is this file supposed to be deleted?

Reply:
No, but it keeps showing up every time I make a new PR. I'll revert it...

Reply:
I asked because I had the same problem locally and I could not understand why

Reply:
I think it might've just been a file that got deleted but the chance never made it to master. It's probably happening for everyone 👀